lib/elixir/lib/task.ex

```diff
@@ -187,7 +187,18 @@ defmodule Task do
   Awaits a task reply.
 
   A timeout, in milliseconds, can be given with default value
-  of `5000`. In case the task process dies, this function will
+  of `5000`. In the event of a timeout handle it with try 
+  catch statement. 
+  
+      timeout = 10
+      try do
+        Task.await(search_task, timeout)
+      catch
```

Comment:
The messages/links/monitors used by a Task are documented and so I thought it was not required to use `Task.await/1,2` or `Task.find/2`. Perhaps this could be made clearer and an example showing a custom await function.

I am not keen on this example because the task is left running and is no longer monitored.